ClassLoader/src/main/java/company/Main.java

```diff
@@ -0,0 +1,11 @@
+package company;
+
+public class Main {
+
+    public static void main(String[] args) {
+        System.out.println(String.class.getClassLoader());
```

Comment:
Didn't get why we need `System.out.println` here. Please use logging or just remove these lines